packages/manager/modules/vps/src/veeam/vps-veeam.routing.js

```diff
@@ -9,13 +10,40 @@ export default /* @ngInject */ ($stateProvider) => {
       },
     },
     resolve: {
+      automatedBackupOption: /* @ngInject */ (
+        $transition$,
+        hasAutomatedBackupOption,
+        VpsService,
+      ) =>
+        hasAutomatedBackupOption
+          ? VpsService.getVeeamInfo($transition$.params().serviceName)
+          : null,
+      breadcrumb: /* @ngInject */ ($translate) =>
+        $translate.instant('vps_veeam'),
       goToVeeamRestore: /* @ngInject */ ($state) => (restorePoint) =>
         $state.go('vps.detail.veeam.restore', { restorePoint }),
       goToVeeamMount: /* @ngInject */ ($state) => (restorePoint, mount) =>
         $state.go('vps.detail.veeam.mount', { restorePoint, mount }),
+      goToVeeamScheduleBackup: /* @ngInject */ ($state) => () =>
+        $state.go('vps.detail.veeam.schedule-backup'),
+      goToVpsUpgrade: /* @ngInject */ ($state) => () =>
+        $state.go('vps.detail.dashboard.configuration.upgrade'),
       goBack: /* @ngInject */ ($state) => () => $state.go('vps.detail.veeam'),
-      breadcrumb: /* @ngInject */ ($translate) =>
-        $translate.instant('vps_veeam'),
+      hasAutomatedBackupOption: /* @ngInject */ ($transition$, VpsService) =>
+        VpsService.getVpsOption(
+          $transition$.params().serviceName,
+          VPS.OPTIONS.AUTOMATED_BACKUP,
+        )
+          .then((vps) => vps.state === VPS.OPTION_STATES.SUBSCRIBED)
+          .catch(() => false),
+      isAutomateBackupAvailable: /* @ngInject */ (ovhFeatureFlipping) => {
+        const automateBackup = 'vps:automateBackup';
+        return ovhFeatureFlipping.checkFeatureAvailability(automateBackup).then(
+          (automateBackupFeature) => true, // automateBackupFeature.isFeatureAvailable(automateBackup),
```

Comment:
Need 2API ?

Reply:
I already implement it. It's available for all regions (approved).
I add it as a good practice.

But i will remove the `true` and use the comment snippet code.

Reply:
Fixed.